consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/ConsistencyChecker.java

```diff
@@ -202,7 +202,9 @@ public void run() {
             findRequest.where(and(
                     withValue("configurationName = " + configuration.getConfigurationName()),
                     withValue("whenAvailableDate <= " + ClientConstants.getDateFormat().format(new Date())),
-                    not(withSubfield("jobExecutions", withValue("completedFlag = true")))));
+                    not(withSubfield("jobExecutions", withValue("jobStatus = 'COMPLETED_SUCCESS'"))),
+                    not(withSubfield("jobExecutions", withValue("jobStatus = 'COMPLETED_PARTIAL'")))
```

Comment:
I think you're right, thanks!  Let me go with some scenarios since it's confusing me too:

### Scenarios:
Each scenario's condistions are AND.  Expectation noted for each scenario.

#### future dated
* whenAvailableDate > new Date()

Processable? No.

#### available and never processed
* whenAvailableDate <= new Date()
* jobExecutions is an empty array

Processable? Yes!

#### available and processed
* whenAvailableDate <= new Date()
* jobExecutions at least one jobStatus equal to 'COMPLETED_SUCCESS' or 'COMPLETED_PARTIAL'

Processable? No.

#### available and currently processing
* whenAvailableDate <= new Date()
* jobExecutions is not empty
* jobExecution has no jobStatus equal to 'COMPLETED_SUCCESS' or 'COMPLETED_PARTIAL'
* max(jobExecutions.actualStartDate) >= new Date() - expectedExecutionMilliseconds)

Processable? Yes!

#### available, currently processing, but presumed "dead"
* whenAvailableDate <= new Date()
* jobExecutions is not empty
* jobExecution has no jobStatus equal to 'COMPLETED_SUCCESS' or 'COMPLETED_PARTIAL'
* max(jobExecutions.actualStartDate) < new Date() - expectedExecutionMilliseconds)

Processable? No.

### Query:
Therefore the query would be:
```
and:
    whenAvailableDate <= new Date()
    or:
        jobExecutions# = 0
        not:
            jobExecutions.jobStatus $in ['COMPLETED_SUCCESS', 'COMPLETED_PARTIAL']
        and:
            not: 
                jobExecutions.jobStatus $not_in [COMPLETED_SUCCESS', 'COMPLETED_PARTIAL']
            jobExecutions.actualStartDate < (new Date() - expectedExecutionMilliseconds)
```